bin/spring

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
why you removed this file?

Reply:
because for heroku there was an old version